CHANGES.md

```diff
@@ -1,5 +1,23 @@
 # Changes
 
+## edge-22.5.3
+
+This edge release fixes a few proxy issues, improves the upgrade process, and
+introduces proto retries to Service Profiles. Also included are updates to the
+bash scripts to ensure that they follow best practices.
+
+* Polished the shell scripts (thanks @joakimr-axis)
+* Introduced retries to Service Profiles by adding an isRetryable function to
+  the proto definition (thanks @mahlunar)
```

Comment:
slightly nitpicky, but I think the PR just made Service Profiles generated from protobufs retryable by default based on the method's idempotency option. Could perhaps re-word a little bit to make it clearer it'll happen by default only for idempotent routes/methods.